addons/binding/org.openhab.binding.dsmr/src/main/java/org/openhab/binding/dsmr/handler/DSMRBridgeHandler.java

```diff
@@ -0,0 +1,220 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.dsmr.handler;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.smarthome.config.core.Configuration;
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.BaseBridgeHandler;
+import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.openhab.binding.dsmr.internal.device.DSMRDevice;
+import org.openhab.binding.dsmr.internal.device.DSMRDeviceConfiguration;
+import org.openhab.binding.dsmr.internal.device.DSMRDeviceConstants;
+import org.openhab.binding.dsmr.internal.device.DSMRDeviceConstants.DeviceState;
+import org.openhab.binding.dsmr.internal.device.DSMRDeviceStateListener;
+import org.openhab.binding.dsmr.internal.discovery.DSMRMeterDiscoveryListener;
+import org.openhab.binding.dsmr.internal.discovery.DSMRMeterDiscoveryService;
+import org.openhab.binding.dsmr.internal.meter.DSMRMeter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link DSMRBridgeHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author M. Volaart - Initial contribution
+ * @since 2.1.0
+ */
+public class DSMRBridgeHandler extends BaseBridgeHandler implements DSMRDeviceStateListener {
+    // logger
```

Comment:
I agree that is does not add any additional information :) I removed those comments